### ISSUE TITLE
Wrong annotation path on Maven project

### ISSUE THREAD

**Issue by kassovix** [goober/sonarqube-pullrequest-decorator-plugin#6]
When decorating a Maven project PullRequest, the annotation path is set to project name (_wopi-core_ in this example):
```
    {
      "reportKey": "com.github.goober.sonarqube.plugin.decorator",
      "externalId": "AXFjQ3PaPs6sIuLymv_G",
      "line": 300,
      "link": "https://<sonarUrl>/project/issues?id=fr.kosmos.wopi:wopi-core&pullRequest=5&open=AXFjQ3PaPs6sIuLymv_G",
      "message": "',' is not followed by whitespace.",
      "path": "wopi-core",
      "severity": "LOW",
      "type": "CODE_SMELL"
    }
```
In Sonar API, this issue has following payload:
```{
      "key": "AXFjQ3PaPs6sIuLymv_G",
      "rule": "checkstyle:com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck",
      "severity": "MINOR",
      "component": "fr.kosmos.wopi:wopi-core:src/main/java/fr/kosmos/wopi/web/WebAbstractionService.java",
      "project": "fr.kosmos.wopi:wopi-core",
      "line": 300,
      "hash": "53a1de7f43f3cd86c88cb228d07e9504",
      "textRange": {
        "startLine": 300,
        "endLine": 300,
        "startOffset": 0,
        "endOffset": 111
      },
      "flows": [],
      "status": "OPEN",
      "message": "',' is not followed by whitespace.",
      "effort": "2min",
      "debt": "2min",
      "assignee": "prenom.nom",
      "tags": [],
      "creationDate": "2019-10-11T13:48:10+0200",
      "updateDate": "2020-04-10T10:44:00+0200",
      "type": "CODE_SMELL",
      "organization": "default-organization",
      "pullRequest": "5",
      "fromHotspot": false
    },
```
So the right path is the last part of `component` key, instead of second part used when mapping this property here: https://github.com/goober/sonarqube-pullrequest-decorator-plugin/blob/4ffc09a01d494329cb4347f35711544f18997ad8/src/main/java/com/github/goober/sonarqube/plugin/decorator/sonarqube/model/Issue.java#L31

**Comment by kassovix** [comment]
I did not find any documentation but the code comment seems clear here:https://github.com/SonarSource/sonarqube/blob/cc26f2f76379066d93d58fa90288d7d6a2a3334c/sonar-plugin-api/src/main/java/org/sonar/api/issue/Issue.java#L117

I think we should use the part after last `:`. I could do a PR with this if you are OK @goober ?

**Comment by goober** [comment]
Thank you for investigating this issue. A pull request would be much appreciated

**Comment by goober** [comment]
Closed by #7
